electrumx/server/session.py

```diff
@@ -1037,8 +1049,10 @@ async def _notify_sessions(self, height, touched):
                 del cache[hashX]
             for hashX in set(op_cache).intersection(touched):
                 del op_cache[hashX]
-                background_task = asyncio.create_task(self.get_history_op(hashX, 10, 0, None, False))
-                await background_task
+                if self.notified_height == await self.daemon.height():
```

Comment:
This code will cause memory leak, so i add `self.notified_height == await self.daemon.height()`. not every height will do `get_history_op` function for cache